components/Post/post.js

```diff
@@ -72,27 +72,18 @@ function Post({ nft, charge }) {
         },
     };
 
-    // const [charge, setCharge] = useState();
+    const [currentCharge, setCurrentCharge] = useState()
+    useEffect(() => {
+        async function getCharge() {
+            var res = await nearState.cnftContract
+                .get_charge({ token_id: nft.token_id.toString() })
```

Comment:
Great Job! I am getting `get_charge` function Not found

Reply:
Yes, the new .env file points to a new contract address